pkg/scheduler/flavorassigner/flavorassigner_test.go

```diff
@@ -1895,39 +1757,37 @@ func TestAssignFlavors(t *testing.T) {
 						Resource(corev1.ResourceCPU, "12").
 						FlavorQuotas,
 				).Cohort("test-cohort").ClusterQueue,
-			cohortResources: &cohortResources{
-				usage: resources.FlavorResourceQuantities{
-					{Flavor: "one", Resource: corev1.ResourceCPU}: 10_000,
-				},
-				requestableResources: resources.FlavorResourceQuantities{
-					{Flavor: "one", Resource: corev1.ResourceCPU}: 12_000,
-					{Flavor: "two", Resource: corev1.ResourceCPU}: 12_000,
-				},
+			secondaryClusterQueue: utiltesting.MakeClusterQueue("test-secondary-clusterqueue").
+				ResourceGroup(
+					utiltesting.MakeFlavorQuotas("one").
+						Resource(corev1.ResourceCPU, "12").
+						FlavorQuotas,
+				).
+				Cohort("test-cohort").
+				Obj(),
+			secondaryClusterQueueUsage: resources.FlavorResourceQuantities{
+				{Flavor: "one", Resource: corev1.ResourceCPU}: 10_000,
 			},
 			wantRepMode: Fit,
-			wantAssignment: Assignment{
-				PodSets: []PodSetAssignment{{
-					Name: "main",
-					Flavors: ResourceAssignment{
-						corev1.ResourceCPU: {Name: "two", Mode: Fit, TriedFlavorIdx: -1},
-					},
-					Requests: corev1.ResourceList{
-						corev1.ResourceCPU: resource.MustParse("12"),
-					},
-					Count: 1,
-				}},
-				Usage: resources.FlavorResourceQuantities{
-					{Flavor: "two", Resource: corev1.ResourceCPU}: 12_000,
-				},
-			},
+			wantAssignment: *MakeAssignment().
+				WithPodSet(
+					*MakePodSetAssignment("main", 1).
+						WithFlavors(ResourceAssignment{
+							corev1.ResourceCPU: {Name: "two", Mode: Fit, TriedFlavorIdx: -1},
+						}).
+						WithRequests(corev1.ResourceList{
+							corev1.ResourceCPU: resource.MustParse("12"),
+						}).
+						Obj(),
+				).
+				WithFlavorResource(resources.FlavorResource{Flavor: "two", Resource: corev1.ResourceCPU}, 12_000).
+				Obj(),
 		},
 	}
 	for name, tc := range cases {
 		t.Run(name, func(t *testing.T) {
 			ctx, _ := utiltesting.ContextWithLog(t)
-			if tc.disableLendingLimit {
-				features.SetFeatureGateDuringTest(t, features.LendingLimit, false)
-			}
+			features.SetFeatureGateDuringTest(t, features.LendingLimit, tc.enableLendingLimit)
```

Comment:
LendingLimit feature is enabled by default, so I believe it was correct.

Reply:
Correct, though this way it's still valid

Reply:
I think I will revert it as there is no effect, no test is setting it to disabled